File: dart/liner_search.dart
void main() {
  List<int> arr = [1, 2, 3, 4, 5, 6];
  int element = 3;

  if (search_element(arr, element)) {
    print("Element $element found");
  } else {
    print("Element $element not found");
  }
}

bool search_element(List arr, int s_element) {
  bool result = false;
  for (int i = 0; i < arr.length; i++) {
    if (s_element == arr[i]) {
      result = true;
      break;
    }
  }
  return result;
}

File: js/liner_search.js
const arr=[1,2,4,5,6,7,8,9,10];
var element=0;
result=search_element(arr,element);
if(result!=-1)
console.log("Element "+element+" found on index "+result);
else
console.log("Element "+element+" not found ");

function search_element(arr,element){
   var result=-1; 
   for(var i=0;i<arr.length;i++){
     if(element==arr[i]){
          result=i;
          break;
     }
   }
   return result;
}